src/test/javascript/spec/component/overview/exercise-details/exercise-details-student-actions.component.spec.ts

```diff
@@ -157,14 +143,13 @@ describe('ExerciseDetailsStudentActionsComponent', () => {
         fixture.detectChanges();
         tick();
 
-        expect(comp.participationStatusWrapper()).toEqual(ParticipationStatus.UNINITIALIZED);
         expect(startExerciseStub).toHaveBeenCalledOnce();
         participationSubject.next(initPart);
 
         fixture.detectChanges();
         tick();
 
-        expect(comp.participationStatusWrapper()).toEqual(ParticipationStatus.INITIALIZED);
+        expect(comp.studentParticipation?.initializationState).toEqual(InitializationState.INITIALIZED);
```

Comment:
could be `expect(comp.exercise.studentParticipations[0].initializationState).toEqual...`
but I am not sure if that is intended behaviour